wcivf/apps/people/managers.py

```diff
@@ -63,20 +63,23 @@ def update_or_create_from_ynr(self, person):
 
         if person['memberships']:
             for membership in person['memberships']:
+                election = None
 
                 if membership['election']:
-                    election, _ = Election.objects.update_or_create(
-                        slug=membership['election']['id'],
-                        name=membership['election']['name'],
-                    )
+                    election = Election.objects.get(
+                        slug=membership['election']['id'])
                     elections.append(election)
 
                 if membership['post']:
                     post, _ = Post.objects.update_or_create(
                         ynr_id=membership['post']['id'],
                         label=membership['post']['label'],
                     )
+                    if election:
+                        post.election = election
```

Comment:
That creates quite a lot more work right now, and I'd rather just get this fix deployed so we're not showing wrong candidates, and then work on doing it properly when I'm more with it / can pair with someone…